Translated/Story/01 New Life in ARKS/msc_011450.csv

```diff
@@ -8,21 +8,21 @@ msc_011450_0070#0,"""Components?"""
 msc_011450_0080#0,"""Oops..."""
 msc_011450_0090#0,"""...Right. I suppose I'll tell you about it."""
 msc_011450_0100#0,"""What's this... ?"""
-msc_011450_0110#0,"""It's a secret weapon we've been developing specifically for fighting Dark Falz."""
+msc_011450_0110#0,"""It's a secret weapon we've been developing specifically for combating Dark Falz."""
```

Comment:
It was fine as "fighting dark falz". Making it say combating Dark Falz feels rather stiff.